src/lib/dnssd/Resolver_ImplMinimalMdns.cpp

```diff
@@ -230,7 +230,9 @@ void PacketParser::ParseSRVResource(const ResourceData & data)
         return;
     }
 
+#if CHIP_MINMDNS_HIGH_VERBOSITY
     ChipLogError(Discovery, "Insufficient parsers to process all SRV entries.");
```

Comment:
Can you please also turn down the following on A records seen on IPv6 only:

```
[[33mSTDOUT[0m][100892:100892] CHIP:DIS: DNSSD parse error: ../../examples/all-clusters-app/linux/third_party/connectedhomeip/src/lib/dnssd/IncrementalResolve.cpp:267: CHIP Error 0x0000002F: Invalid argument
```
?

Reply:
Done. Delta is slightly larger now.